### PR TITLE
Use version helper function more.

### PR DESCRIPTION
**Pull request by peterwilsoncc**
Trac ticket: https://core.trac.wordpress.org/ticket/61627

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/includes/class-wp-site-health.php [hunk 1]
```diff
@@ -289,7 +289,7 @@ public function get_test_wordpress_version() {
 		} else {
 			foreach ( $core_updates as $core => $update ) {
 				if ( 'upgrade' === $update->response ) {
-					$current_version = explode( '.', $core_current_version );
+					$current_version = explode( '.', wp_get_wp_version() );
```
Comment (hellofromtonya) [inline src/wp-admin/includes/class-wp-site-health.php]: @peterwilsoncc why change it here and not where `$core_current_version` is assigned?

```
$core_current_version = get_bloginfo( 'version' );
```

File: src/wp-admin/includes/update-core.php [hunk 2]
```diff
@@ -1538,12 +1538,12 @@ function update_core( $from, $to ) {
...
- * @global string             $wp_version          The WordPress version string.
  *
  * @param string $to Path to old WordPress installation.
  */
 function _preload_old_requests_classes_and_interfaces( $to ) {
-	global $_old_requests_files, $wp_filesystem, $wp_version;
+	global $_old_requests_files, $wp_filesystem;
+	$wp_version = wp_get_wp_version();
 
 	/*
```
Comment (hellofromtonya) [inline src/wp-admin/includes/update-core.php:1549]: Hey @peterwilsoncc, should the global also be changed in `_redirect_to_about_wordpress()`?

File: src/wp-admin/includes/class-wp-site-health.php (hunk 1 above)
Comment (peterwilsoncc) [reply src/wp-admin/includes/class-wp-site-health.php]: No idea why I did it differently here, I've fixed this in f7416853952d7f1208f9d9cf6b80029fe81f519b

File: src/wp-admin/includes/update-core.php (hunk 2 above)
Comment (peterwilsoncc) [reply src/wp-admin/includes/update-core.php:1549]: Yep, done in 1c9ed13f2ca47a57551045a03904c1b4874126ea.

Thanks for the review.

**Review by hellofromtonya — APPROVED** [review]
Great job @peterwilsoncc. LGTM 👍  Ready for commit

**Comment by peterwilsoncc** [comment]
[r59159](https://core.trac.wordpress.org/changeset/59159) / https://github.com/WordPress/wordpress-develop/commit/03834fec6848fe433045a3b19b1b45236140c3e4

**Pull request closed without merging**